hack/bump-zversions.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -e
+
+BASEDIR="$(dirname $0 )"
+
+# shellcheck source=common.sh
+source "$(dirname $0)/common.sh"
+
+CSV_VERSION_A=( ${CSV_VERSION//./ } ) # replace points, split into array
+ZVERSION_NOW="${CSV_VERSION_A[2]}"
+ZVERSION_NEXT=$( expr ${ZVERSION_NOW} + 1)
+CSV_VERSION_NEXT="${CSV_VERSION_A[0]}.${CSV_VERSION_A[1]}.${ZVERSION_NEXT}"
+
```

Comment:
I expect that the PR will update the variable under the `hack/common.sh` and after will call to relevant scripts
- call https://github.com/openshift-kni/performance-addon-operators/pull/520/files#diff-cb3ab62f5a92ed6c60ec87fc8fff45dc8ef7f17287ad3a24c6ebf7384cd91fe9R46
- call to `make generate` (I do not see an update of the skip range)
- update bundle images

Reply:
sure, why not? let me make this script smarter
with one question though: how we can update the bundle images outside CI?

Reply:
ah sorry for misleading you I meant to update bundle Dockerfiles

Reply:
ok, so we want to update just one value in one place and autogenerate everything else, right? This will require much more changes.